src/models/key.model.ts

```diff
@@ -3,11 +3,13 @@ import { KeyInfo } from "../types";
 export class Key {
   public name: string;
   public ascii: string;
-  public audio: string;
+  public audio: HTMLAudioElement;
+  public element: HTMLDivElement;
```

Comment:
model should not hold any ui element in it including audio. It only should maintain plain data from which these elements can be constructed